webhook_to_fedora_messaging/config/__init__.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from os import environ
+
+import tomllib
```

Comment:
The `tomllib` module has been added in Python 3.11, if you want to support earlier version you'll have to add a dependency on `tomli`. Otherwise, please change the minimum version in `pyproject.toml`.

Reply:
I lifted up the base Python version requirement to be 3.11.